src/algorithms/digi/SiliconTrackerDigiConfig.h

```diff
@@ -11,7 +11,7 @@ namespace eicrecon {
     // sub-systems should overwrite their own
     // NB: be aware of thresholds in npsim! E.g. https://github.com/eic/npsim/pull/9/files
     double threshold  = 0 * dd4hep::keV;
-    double timeResolution = 8;   /// TODO 8 of what units??? Same TODO in juggler. Probably [ns]
+    double timeResolution = 8;   /// same unit as sim_hit.getTime, probably [ns]
```

Comment:
The plan is to have eventually to everywhere use
```c++
#include <edm4eic/unit_system.h>
```
And it is already appropriate here:
```suggestion
    double timeResolution = 8 * edm4eic::unit::ns;
```
https://github.com/eic/EDM4eic/blob/0bd6edd97d0b28b30605ab90dcf338a3244d16dd/utils/include/edm4eic/unit_system.h#L32